### PR TITLE
Update gtfs

### PR DESCRIPTION
**Pull request by markmeyerphoto**
This allows the bus tracker to read GTFS data directly from the raw text files, which avoids the need to process the files into JSON format. It watches the files for changes and re-reads them when the files have changed. 

It periodically checks the muni for a new GTFS zip file (currently at 2:01 am daily). It there's a new file, it will download it and unzip it.

### CODE REVIEW

**Comment by markmeyerphoto** [comment]
The setInterval created by node-cron was causing the nodeunit tests to never exit. I exported the cron object and manually called stop in the test teardown. This seems to work, but I'm not sure if there's a better way to do this.

**Comment by NigelKibodeaux** [comment]
How about putting the cron job setup line in /bin/www? I think it fits there because that's where the web server is started and it has the added benefit of not being run by the tests.

File: lib/gtfs.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+class WatchedFile {
+    // This class will read and process the file passed in the constructor and watch it
+    // on change it will re-read and re-process the file. 
+    // Processed data available in .parsed_data by default
+    constructor(filename) {
+        this.filename = filename;
+        this.parsed_data = {}
+        if (fs.existsSync(this.filename)) {
+            this.readfile({first_run: true});
```
Comment (markmeyerphoto) [inline lib/gtfs.js]: I should fix this — been going back and forth between this and another project — thought I was passing a Python dictionary.

**Comment by brendanbabb** [comment]
Tested on test AWS and ignoring Twilio account SID

**Pull request merged**